### ISSUE TITLE
Save Email template and Landing page doesn't work!

### ISSUE THREAD

**Issue by meszicsaba84** [gophish/gophish#2610]
**Thanks for reaching out! We're happy to help resolve issues as quickly as possible.**

**Please use this template when creating a new issue. If you do not follow this template format, your issue may be closed without comment.**

Before filing a new issue, please use the search bar at the top of the browser to search for similar issues. Also, please make sure you have read our [documentation](https://getgophish.com/documentation/), which covers how to use Gophish.

What version of Gophish are you using?: gophish@v0.12.1/ 

Brief description of the issue: When I would like to create a new email template (with Importing source from Gmail) and I would like to save it nothing happens.

What are you expecting to see happen? : To save my email template into my templates on the GUI.

What are you seeing happen? Nothing happens and I attach what I can see in the CLI on the server (Ubuntu).

If this question is related to email templates or landing pages not working as expected, please provide your template or landing page below:

```html
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><html style="background: #ededed;" xmlns="http://www.w3.org/1999/xhtml"><head>
	<title>Eaton</title><!--add Hotmail fixes, background colour in Hotmail is specificed on .ExternalClass-->

	<link rel="icon" href="http://images.electricalsector.eaton.com/EloquaImages/clients/EatonElectrical/%7be91c9279-3776-4896-9040-0c293397d992%7d_favicon.png"/>
	<meta content="text/html; charset=UTF-8" http-equiv="Content-Type"/>
	<style type="text/css">
	html {
		background-color: #ededed;
		height:100%;
	}
	body {
		-ms-text-size-adjust: 100%;
		-webkit-text-size-adjust: 100%;
		margin: 0;
		padding: 0;
		width: 100% !important;
		min-height: 100%;
	}
	.ExternalClass {
		width: 100% !important;
	}
	
	.ExternalClass,
	.ExternalClass p,
	.ExternalClass span,
	.ExternalClass font,
	.ExternalClass td,
	.ExternalClass div {
		line-height: 100%;
	}

    /*RESPONSIVE CODE*/
    @media screen and (max-width: 640px) {
        *[class=emailwrapto100pc] {
            height: auto !important;
            width: 100% !important;
        }

        *[class=emailwrapto100pcwhite] {
            height: auto !important;
            width: 100% !important;
        }

        *[class=emailcolsplit] {
            float: left !important;
            width: 100% !important;
        }

        *[class=mobilecta] {
            background-color: #afafaf;
            border-radius: 4px;
            color: #ffffff !important;
            display: block;
            font-size: 12px !important;
            font-weight: bold;
            height: 34px;
            line-height: 30px !important;
            padding-bottom: 5px;
            padding-top: 5px;
            width: 260px !important;
        }

        *[class=emailshowmob] {
            display: table-cell !important;
        }

        *[class=mobilebannerwidth] {
            width: 100% !important;
        }

        *[class=hero] {
            width: 100% !important;
        }

        /*CTM mobile styles*/
        *[class=emailnomob] {
            width:0 !important;
        }
    }


    </style>
	<!--this specifies the background colour for webmail clients that don't recognise the body tag-->
	<!--this specifies the background image and colour in Outlook & other desktop email clients-->
</head>
<body bgcolor="#ededed" style="background:#ededed;" style="PADDING-BOTTOM: 0px; MARGIN: 0px; PADDING-LEFT: 0px; PADDING-RIGHT: 0px; PADDING-TOP: 0px;height:100%;width:100%;">




	<table width="100%" cellspacing="0" cellpadding="0" border="0" style="BACKGROUND: #ededed" bgcolor="#ededed">
		<tbody>
		<tr>
			<td align="center" class="">
				<table width="650" cellspacing="0" cellpadding="0" border="0" class="emailwrapto100pc">
					<tbody>
					<tr>
						<td width="100%" colspan="5" align="right" style="FONT-FAMILY: arial,sans-serif; FONT-SIZE: 10px; padding-right: 15px;padding-bottom:10px;padding-top:15px;">
								<a href="{{.URL}}" style="font-family: Arial, Helvetica, sans-serif; color: #535353; font-size: 11px; text-decoration: none; line-height: 16px;" target="_blank"><u>View this email online</u></a>
						</td>
					</tr>
					<tr>
						<td width="100%" bgcolor="#ffffff" colspan="5" style="BORDER-LEFT: #ededed 1px solid; BORDER-RIGHT: #ededed 1px solid" class=""> </td>
					</tr>
					<tr bgcolor="#FFFFFF">
						<td width="15" bgcolor="#ffffff" style="BORDER-LEFT: #ededed 1px solid"> </td>
						<td align="left">
							<table cellpadding="0" cellspacing="0" border="0" align="left">
								<tbody><tr>
									<td style="line-height:16px;"><img width="1" height="16" style="display: block;" src="http://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif"/></td>
								</tr>
								<tr>
									<td width="498" bgcolor="#ffffff" style="" align="left"><img width="170" height="59" border="0" src="http://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7Bf7876aee-2346-4f06-bcf2-474ba1b15bef%7D_eaton.gif" style="display:block;"/></td>
								</tr>
								<tr>
									<td style="line-height:16px;"><img width="1" height="16" style="display: block;" src="http://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif"/></td>
								</tr>
							</tbody></table>
						</td>
						<td width="63" bgcolor="#ffffff" class=""> </td>
						<td width="59" bgcolor="#ffffff"> </td>
						<td width="15" bgcolor="#ffffff" style="BORDER-RIGHT: #ededed 1px solid"> </td>
					</tr>
					<tr>
						<td width="15" bgcolor="#ffffff" style="BORDER-LEFT: #ededed 1px solid"> </td>
						<td width="620" bgcolor="#ffffff" colspan="3" style="BORDER-BOTTOM: #ededed 3px solid" class=""> </td>
						<td width="15" bgcolor="#ffffff" style="BORDER-RIGHT: #ededed 1px solid"> </td>
					</tr>
					<tr>
						<td width="100%" bgcolor="#ffffff" aligh="left" colspan="5" style="BORDER-LEFT: #ededed 1px solid; BORDER-RIGHT: #ededed 1px solid">
							<table width="100%" cellspacing="0" cellpadding="0" border="0" align="left">
								<tbody>
								<tr>
									<td align="left" style="font-family: arial,sans-serif; color: #444; padding-left: 15px; font-size: 12px; padding-right: 15px; border-style: none; padding-bottom:15px;" class="">
										<br/>
Dear Csaba,
										<br/>Thank you for your continued interest in receiving Eaton communications.<br/>
										<br/>

										To ensure we only send information that is most relevant to you, please visit your <a href="{{.URL}}" target="_blank" style="COLOR: #0072b6">personalised Eaton resource centre</a> to tell us a little more about you and to select your personal interest areas.

										<br/><br/>

										By accessing the resource centre, you are confirming your continued choice to receive communications from Eaton.<br/><br/>

										<strong>Your Eaton Team</strong><br/>
										<a href="{{.URL}}" target="_blank" style="COLOR: #0072b6">EMEAMarketingAutomation@Eaton.com</a><br/><br/>
									</td>
								</tr>
								</tbody>
							</table>
						</td>
					</tr>

					</tbody>
				</table>
			</td>
		</tr>
		</tbody>
	</table>


<div style="BACKGROUND-COLOR: #f0f0f0" width="100%">
<table align="center" bgcolor="#5B6770" border="0" cellpadding="0" cellspacing="0" class="emailwrapto100pc" style="color:#fff;" width="640">
	<tbody>
		<tr>
			<td align="center" colspan="5" height="14" style="background-color: #005EB8; LINE-HEIGHT: 1px; FONT-SIZE: 1px; padding-top:10px; padding-bottom:10px">
			<table border="0" cellpadding="10" cellspacing="0" class="fullwidth" style="width:320px;" width="340">
				<tbody>
					<tr>
						<td>
						<table align="left" border="0" cellpadding="0" cellspacing="0" class="fullwidth">
							<tbody>
								<tr>
									<td align="center">
									<p class="text-padding-bottom" style="font-family: Arial,Sans-serif; font-size:14px; line-height:16px; color:#ffffff; padding:0; margin:6px 0 0 0;font-weight:bold;">Connect with Eaton:</p>
									</td>
								</tr>
							</tbody>
						</table>

						<table align="right" border="0" cellpadding="0" cellspacing="0" class="fullwidth">
							<tbody>
								<tr>
									<td align="center" style="padding:0 10px 0;font-size: 1px; line-height: 1px;"><a href="{{.URL}}" target="_blank"><img alt="linkedin" class="social" height="30" src="https://images.electricalsector.eaton.com/EloquaImages/clients/EatonElectrical/{b34ad559-3a3c-4265-aa2e-7fa1ad4bf9f0}_social-linkedin.jpg" style="display:block; font-family:Arial, Sans-serif; font-size:14px; line-height:19px;line-height:1px;" width="30"/></a></td>
									<td align="center" style="padding:0 10px 0;font-size: 1px; line-height: 1px;"><a href="{{.URL}}" target="_blank"><img alt="twitter" class="social" height="30" src="https://images.electricalsector.eaton.com/EloquaImages/clients/EatonElectrical/{a2eb2fcf-ee2c-4a4b-844c-065f080c9057}_social-twitter.jpg" style="display:block; font-family:Arial, Sans-serif; font-size:14px; line-height:19px;line-height:1px;" width="30"/></a></td>
									<td align="center" style="padding:0 10px 0;font-size: 1px; line-height: 1px;"><a href="{{.URL}}" target="_blank"><img alt="youtube" class="social" height="30" src="https://images.electricalsector.eaton.com/EloquaImages/clients/EatonElectrical/{f7dc69f1-c251-4ce2-835a-71bd7013894c}_social-youtube.jpg" style="display:block; font-family:Arial, Sans-serif; font-size:14px; line-height:19px;line-height:1px;" width="30"/></a></td>
								</tr>
							</tbody>
						</table>
						</td>
					</tr>
				</tbody>
			</table>
			</td>
		</tr>
		<tr>
			<td colspan="5" height="14" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="14" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
		</tr>
		<tr>
			<td width="12"><img height="1" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="12"/></td>
			<td class="emailnomob" width="12"><img height="1" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="12"/></td>
			<td align="center" valign="top">
			<table border="0" cellpadding="0" cellspacing="0" width="100%">
				<tbody>
					<tr>
						<td>
						<table cellpadding="0" cellspacing="0" width="100%">
							<tbody>
								<tr>
									<td height="10" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="10" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
								</tr>
								<tr>
									<td>
									<center><img src="https://images.electricalsector.eaton.com/EloquaImages/clients/EatonElectrical/%7be0e22757-8973-4936-aa22-e7b9343b8066%7d_eaton-logo-footer.jpg"/></center>
									</td>
									<td><img height="14" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block; width:25px;" width="20"/></td>
									<td align="right" class="emailcolsplit" style="LINE-HEIGHT: 15px; FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff;); FONT-SIZE: 11px; text-align:left;" valign="top">Eaton is a power management company.<br/>
									We provide energy-efficient, hydraulic, electrical and mechanical power more efficiently, safely and sustainably.</td>
								</tr>
							</tbody>
						</table>
						</td>
					</tr>
					<tr>
						<td class="padding10top" height="16" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px; border-bottom:1px solid #ccc; width:100%;"> </td>
					</tr>
					<tr>
						<td class="padding10top" height="16" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="16" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
					</tr>
					<tr>
						<td align="left" class="mobilelink2" style="LINE-HEIGHT: 15px; FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR:#fff;); FONT-SIZE: 11px; text-align:left;" valign="top">This email was sent to meszicsaba@gmail.com by Eaton. Links in this e-mail are tracked for marketing purposes to ensure we deliver the most relevant content to you.</td>
					</tr>
					<tr>
						<td class="padding10top" height="16" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="16" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
					</tr>
					<tr>
						<td align="left" class="mobilelink2" style="LINE-HEIGHT: 15px; FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR:#fff;); FONT-SIZE: 11px; text-align:left;" valign="top">To ensure that you continue to receive important updates and newsletters, please add <a href="{{.URL}}" style="color: #fff;">uk-marketing@eaton.com</a> to your safe-senders list.</td>
					</tr>
					<tr>
						<td class="padding10top" height="16" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="16" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
					</tr>
					<tr>
						<td align="left" class="mobilelink2" style="LINE-HEIGHT: 14px; FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff; FONT-SIZE: 11px; text-align:left;" valign="top">Eaton Electric Limited, 252 Bath Road, Slough, Berkshire SL1 4DX, United Kingdom</td>
					</tr>
					<tr>
						<td height="16" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="16" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
					</tr>
					<tr>
						<td align="left" class="mobilelink2" style="LINE-HEIGHT: 14px; FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff; FONT-SIZE: 12px" valign="top"><span><a class="mobilelink" href="{{.URL}}" style="FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff; FONT-SIZE: 11px;TEXT-DECORATION: underline" target="_blank">Manage your preferences </a></span> | <span><a class="mobilelink" href="{{.URL}}" style="FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff; FONT-SIZE: 11px;TEXT-DECORATION: underline" target="_blank">Privacy Policy</a></span> | <span> <a class="mobilelink" href="{{.URL}}" style="FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff; FONT-SIZE: 11px;TEXT-DECORATION: underline" target="_blank">Unsubscribe from all channels</a></span> | <span><a class="mobilelink" href="{{.URL}}" style="FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff; FONT-SIZE: 11px;TEXT-DECORATION:underline;" target="_blank">Email us</a></span> | <span><a class="mobilelink" href="{{.URL}}" style="FONT-FAMILY: Arial,Helvetica,sans-serif; COLOR: #fff; FONT-SIZE: 11px;TEXT-DECORATION: underline" target="_blank">www.eaton.com</a></span></td>
					</tr>
					<tr>
						<td height="16" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="16" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
					</tr>
				</tbody>
			</table>
			</td>
			<td width="12"><img height="1" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="12"/></td>
			<td class="emailnomob" width="12"><img height="1" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="12"/></td>
		</tr>
		<tr>
			<td colspan="5" height="14" style="LINE-HEIGHT: 1px; FONT-SIZE: 1px"><img height="14" src="https://images.electricalsector.eaton.com/eloquaimages/clients/EatonElectrical/%7B5b9aa183-b080-4f34-9b61-41e49a0075dd%7D_spacer.gif" style="DISPLAY: block" width="1"/></td>
		</tr>
	</tbody>
</table>
</div>


<img src="https://app.electricalsector.eaton.com/e/FooterImages/FooterImage1?elq=d5794bfc17aa431aa1780b850fcf30cc&amp;siteid=1521" alt="" border="0" width="1px" height="1px"/></body></html>
```

Please provide any terminal output that may be relevant below:

```
Attached
```

Please provide as many steps as you can to reproduce the problem:
 * Step 1 - Create new email template
 * Step 2 - Adding name
 * Step 3 - Import email.
 * Step 4 - go to Gmail, View source, copy to Gophish, Import
 * Step 5 - trying to save.
 
 
![gophish_log](https://user-images.githubusercontent.com/114218758/191918814-6396002e-31f2-4e7c-afa1-d1a6a29a65a3.png)

**Comment by meszicsaba84** [comment]
I can see there is also this error: "2022/09/23 08:27:27 http: TLS handshake error from 176.63.11.202:4309: remote error: tls: unknown certificate" is this related to not saving the template?

**Comment by meszicsaba84** [comment]
Hi, any suggestsions? I have also built my infra again from scratch - but I got the same error... When I click the Save button seems nothing happens on the GUI.

**Comment by meszicsaba84** [comment]
I have also realized that continuous loading (circle) is moving on the E-mail template and Landing pages only on the GUI - but on other pages it is fine. Also when I hit the "Save" button nothing happens on the console, i.e. no POST or GET message.

**Comment by glennzw** [comment]
This issue has been cropping up a lot recently, and I think I've figured out why. When `go get` became deprecated we moved to `go install` - but there's an issue using `go install` with folders called `vendor` as this is a reserved word - so the vendor folder with all the Javascript for rendering charts and editing HTML don't get downloaded (see discussion here https://github.com/golang/go/issues/48332).

The only solution I can think of for now is rather than using `go install` use `git clone https://github.com/gophish/gophish.git`

Alternatively download the precompiled binary.
